### PR TITLE
test cases for the service account

### PR DESCRIPTION
**Pull request by ayesha54**
test cases for the service account for the node observability operator

### CODE REVIEW

File: pkg/operator/controller/names.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nodeobservabilitycontroller
+
+const (
+	// NodeObservabilityBaseName  is the base name for any ExternalDNS resource.
```
Comment (josefkarasek) [inline pkg/operator/controller/names.go:20]: copy&paste error?

**Review by sherine-k — APPROVED** [review]
Thanks for well structured tests @ayesha54 ;)

**Review by josefkarasek — CHANGES_REQUESTED** [review]

**Review by lmzuccarelli — COMMENTED** [review]
if you don't mind fixing the nit: Josef pointed out

File: pkg/operator/controller/names.go (hunk 1 above)
Comment (ayesha54) [reply pkg/operator/controller/names.go:20]: let me fix

**Comment by ayesha54** [comment]
> if you don't mind fixing the nit: Josef pointed out

sure, fixed.

**Pull request merged**